testing/MLDB-180-basic-join.js

```diff
@@ -258,7 +258,7 @@ for (var i in funcs) {
 // MLDB-1088
 
 expected = [
-   [ "_rowName", "test1.x", "test1.y", "test1.z" ],
+   [ "_rowName", "x", "y", "z" ],
```

Comment:
ok, so in some cases a join will lead to the table name in the column name and in some cases it won't?

... I really need to spec out the behaviour we want here :(